en/ui-system/components/editor/canvas.md

```diff
@@ -1,14 +1,14 @@
 # Canvas Component Reference
 
-The **Canvas** component is in charge of rendering all UI components, all UI elements must be placed under a __Canvas__ so that it can be rendered. The **Canvas** component can adapt the UI elements in __design resolution__ to the actual resolution of the device screen and can control zoom of all UI elements. There can be multiple __Canvas__ in the scene, each of them represents a camera, the rendering results of each canvas and 3d cameras can be presented in a certain order controlled by the `priority`.
+![](canvas/canvas.png)
 
-The Canvas __design resolution__ and __adaptation scheme__ are uniformly set in the __Project Setting__. A camera is provided inside the __Canvas__, and the default z axis direction is __-1000 ~ 998__, so the z axis on the UI must be within this range to display properly (without taking the threshold value).
+The node where the **Canvas** component is the root of the UI rendering. There can be multiple __Canvas__ in the scene. **all UI elements must be placed under a __Canvas__ so that it can be rendered**. Under which Canvas node, it is rendered by which Canvas. Each canvas can be presented in a certain order controlled by the `priority` attribute.
```

Comment:
The node where the **Canvas** component is the root of the UI rendering. There can be multiple __Canvas__ in the scene. **all UI elements must be placed under a __Canvas__ so that it can be rendered**. Under which Canvas node, it is rendered by which Canvas. Each canvas can be presented in a certain order controlled by the `priority` attribute.
-->
The node where the **Canvas** component is located is the __root__ of the UI rendering. There can be multiple __Canvas__ nodes in a scene. **All UI elements must be placed under a __Canvas__ node so that it can be rendered**. UI Elements are rendered by the __Canvas__ node that they are located under. Each __Canvas__ can be presented in a certain order controlled by the `priority` attribute.